packages/amplify-cli/src/extensions/amplify-helpers/get-root-stack-id.ts

```diff
@@ -4,8 +4,8 @@ export function getRootStackId(): string {
   const teamProviderInfo = stateManager.getTeamProviderInfo();
   const { envName } = stateManager.getLocalEnvInfo();
   const envTeamProviderInfo = teamProviderInfo[envName];
-  if (envTeamProviderInfo && envTeamProviderInfo.awscloudformation) {
-    const stackId = envTeamProviderInfo.awscloudformation.StackId;
+  const stackId = envTeamProviderInfo?.awscloudformation?.StackId;
+  if (typeof stackId === 'string') {
     return stackId.split('/')[2];
   }
   throw new Error('Root stack Id not found');
```

Comment:
Shall we enhance the error to show what was the value of `stackId` that we considered "not found"?